src/components/TaskCard/index.module.scss

```diff
@@ -0,0 +1,13 @@
+.textarea {
+  width: 21.938rem;
+  height: 5rem;
+  border-radius: 0.313rem;
+  resize: none;
+  border: 0.188rem solid #d1d8ff;
```

Comment:
It is a good practice to maintain a color variables file where all colors can be included.